nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-jetty/src/main/java/org/apache/nifi/web/server/JettyServer.java

```diff
@@ -1012,6 +1016,28 @@ protected static void configureSslContextFactory(SslContextFactory.Server contex
         contextFactory.setIncludeProtocols(TlsConfiguration.getCurrentSupportedTlsProtocolVersions());
         contextFactory.setExcludeProtocols("TLS", "TLSv1", "TLSv1.1", "SSL", "SSLv2", "SSLv2Hello", "SSLv3");
 
+        // on configuration, replace default application cipher suites with those configured
+        final String includeCipherSuitesProps = props.getProperty(NiFiProperties.WEB_HTTPS_CIPHERSUITES_INCLUDE);
+        if (StringUtils.isNotEmpty(includeCipherSuitesProps)) {
+            final String[] includeCipherSuitesRuntime = contextFactory.getIncludeCipherSuites();
+            final String[] includeCipherSuites = includeCipherSuitesProps.split(REGEX_SPLIT_PROPERTY);
+            logger.info("Replacing include cipher suites with configuration; runtime = {}, raw property = {}, parsed property = {}.",
```

Comment:
Do you think it is likely that the raw property will be that different from the parsed property?  It seems like the only difference should be the number of spaces, in which case, logging both doesn't seem very useful.

Reply:
I'm worried about edge cases, but may be overly cautious.  Will change to logging just the parsed value.